### ISSUE TITLE
[FR]: If Browser Layout is Vertical: Automatically open the layout on selecting feed

### ISSUE THREAD

**Issue by AmmarSaleemG** [martinrotter/rssguard#609]
### Brief description of the feature request

If Browser Layout is Vertical: Automatically open the layout on selecting feed, 

#### Performance Issue:
My articles are piling up to 37 thousand, and with vertical browser layout, I am experiencing performance issues.
ONLY first article, when I select it, it opens browser, which takes time to open it. After that, it is smooth.

#### Possible Reason:
> I think, Since Multiline was enabled in #570 and tweaked accordingly as needed. This might be related to it. Because, it might take the application time to parse through every article and shirink them to fit the size of browser. **It is an assumption (which can be wrong).**

#### Possible Fix
The only solution to it can be to pre-(automatically) load the browser layout (when set vertical) with placeholder inside (if we do not want to select first item, whichever is easy to be implemented).

**Comment by akinokonomi** [comment]
Looks similar to https://github.com/martinrotter/rssguard/issues/409 , https://github.com/martinrotter/rssguard/issues/424

**Comment by AmmarSaleemG** [comment]
> Looks similar to #409 , #424

Yes, exactly.

**Comment by martinrotter** [comment]
Do you have multiline setup enabled?

**Comment by martinrotter** [comment]
> My articles are piling up to 37 thousand, and with vertical browser layout, I am experiencing performance issues.
> ONLY first article, when I select it, it opens browser, which takes time to open it. After that, it is smooth.

Also, could you make video? I would rather fix the problem instead of work arounding it.

#424 is on my internal plan to be implemented anyway, but I would like to understand this problem first.

**Comment by AmmarSaleemG** [comment]
What in #424 was said, I said _almost_ the same. **Focus on picture marked red below:**

![image](https://user-images.githubusercontent.com/15247707/150988317-043e08fb-b21a-45f4-9a99-32149fae91bf.png)

- The guy selected the **Feed**, which **automatically** opened the **Message Viewer** containing **description** (_placeholder_)
    - The placeholder here has feed **title** its **description** its **link**, which is an apt way to portray about feed properties.

      - what we can enhance in this is we add **feed count** and **last fetch time**.

The message viewer in this picture is Horizontal, while in my settings, it is **vertical** (_Because I like side by side view better_).

This is all what I said. 
In short, **we want message viewer opened always** either as an option or by default. That's it.

Let me know if you need more info about it.

**Comment by martinrotter** [comment]
OK, in that case this is duplicate.

#424
